packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -7772,13 +7813,12 @@ describe('plugin-meetings', () => {
         it('sets correctly', () => {
           assert.notOk(meeting.permissionTokenPayload);
 
-          const permissionTokenPayloadData = {permission: {userPolicies: {a: true}}, exp: '1234'};
+          const permissionTokenPayloadData = {permission: {userPolicies: {a: true}}, exp: '123456'};
 
-          const jwtDecodeStub = sinon.stub(jwt, 'decode').returns(permissionTokenPayloadData);
-
-          meeting.setPermissionTokenPayload();
+          meeting.setPermissionTokenPayload(
```

Comment:
Has the function really changed to accept the permission token as an argument? From what I see, we should have actually stubbed the new jwtDecode library rather than taking that code out at all.

Reply:
There is no change, jwtDecode is the default export for jwtDecode package, stubbing it is not straight forward. Thats why to run the test just passing a valid dummy encoded data. The data is taken from the code base itself.